src/Microsoft.Security.Utilities.Core/CompiledHighPerformancePattern.Generated.cs

```diff
@@ -32,6 +32,11 @@ partial class CompiledHighPerformancePattern
     /*lang=regex*/ private const string RawRegex5 = """^[a-zA-Z0-9+/]{33}.{4}[A-P][a-zA-Z0-9+/]{5}=""";
     /*lang=regex*/ private const string RawRegex6 = """^[a-zA-Z0-9+/]{76}.{4}[a-zA-Z0-9+/]{5}[AQgw]==""";
     /*lang=regex*/ private const string RawRegex7 = """^[a-zA-Z0-9+/]{42}.{4}[A-D][a-zA-Z0-9+/]{5}""";
+    /*lang=regex*/ private const string RawRegex8 = """^.{3}[a-z2-7]{43}""";
+    /*lang=regex*/ private const string RawRegex9 = """^.{3}[A-Z2-7]{43}""";
+    /*lang=regex*/ private const string RawRegex10 = """^.{4}[0-9a-f\-]{32,34}""";
```

Comment:
>{4}[0-9a-f\-]{32,34} [](http://example.com/codeflow?start=58&length=20)

i'm missing the context for adding this particular pattern, why do we need it? will keep looking to see if I can answer my own question.  my eye caught this pattern initially because there's no similar pattern for uppercase alpha. on surface, this sorta looks like it could be a hex 32-byte value with or without hyphens but if so, the length checks don't make sense (there are 4 hyphens to elide in a GUID).

Reply:
This is the generated codoe for the `AzureDatabricksPat` scoped regex.

I believe it follows from the pre-existing general regex for that correctly. Is there something wrong with that or am I missing a divergence from it?

Reply:
Hmm, looking at SPMI and the test cases here, it seems it should be exactly 32 `[0-9a-f]` followed by optional `-[0-9]` ?

Reply:
Again here, I will leave behavior change out of scope for this PR and file an issue.

Reply:
#197

Reply:
Yes, you got it:

`^.{4}[0-9a-f\-]{32}(-[0-9])?`